MPCAutofill/cardpicker/templates/cardpicker/index.html

```diff
@@ -57,44 +54,44 @@ <h2>MPC Autofill <em class="orpheus" style="font-size: 1rem;">by
         </div>
     </div>
 
-    <div class="row">
-        <div class="col-lg-6">
-            <div style="text-align: center">
-                <a class="btn btn-block btn-info mt-2" style="border-radius: 200px" href="#">Guide</a>
-            </div>
-        </div>
-        <div class="col-lg-6">
-            <input form="cardinput" type="submit" class="btn btn-success mt-2" value="Submit">
-
-            <button type="button" class="btn btn-primary mt-2" data-toggle="modal" data-target="#selectDrivesModal">
-                Search Settings
-            </button>
+    <div class="row justify-content-end">

```

Comment:
looking at the live site compared to the changes i just pushed, i think the only thing you'll need to tweak here is using `outline` styling :)

Reply:
10/4 I think I covered during the the rebase!